### ISSUE TITLE
Parse additional modules in TLA file

### ISSUE THREAD

**Issue by lemmy** [alygin/vscode-tlaplus#185]
Consider the TLA file below, for which parse errors in modules `EWD840_TE` and `TTraceTraceDef` are not detected (they are reported by the Toolbox).

```tla
---- MODULE EWD840_TTrace_1608881116 ----
EXTENDS Toolbox, TLC, EWD840

TTraceExpression ==
    LET EWD840_TE == INSTANCE EWD840_TE
    IN EWD840_TE!TraceExpression

TTraceTraceDef == INSTANCE TTraceTraceDef
def_ov_16088811171232000 == TTraceTraceDef!def_ov_16088811171232000

inv_16088811171243000 ==
    ~(color = ((0 :> "white" @@ 1 :> "white" @@ 2 :> "white")) /\
    tpos = (0) /\
    tcolor = ("white" ) /\
    active = ((0 :> FALSE @@ 1 :> FALSE @@ 2 :> FALSE)))

_SpecTEInit ==
    /\ tpos = _TETrace[1].tpos
    /\ active = _TETrace[1].active
    /\ tcolor = _TETrace[1].tcolor
    /\ color = _TETrace[1].color

_SpecTENext ==
    /\ \E i,j \in DOMAIN _TETrace:
        /\ \/ j = i + 1
        /\ tpos  = _TETrace[i].tpos
        /\ tpos' = _TETrace[j].tpos
        /\ active  = _TETrace[i].active
        /\ active' = _TETrace[j].active
        /\ tcolor  = _TETrace[i].tcolor
        /\ tcolor' = _TETrace[j].tcolor
        /\ color  = _TETrace[i].color
        /\ color' = _TETrace[j].color

=============================================================================

---- MODULE EWD840_TE ----
EXTENDS Toolbox, TLC, EWD840

TraceExpression == 
    [
        tpos |-> tpos
        ,active |-> active
        tcolor |-> tcolor  \* <==== Parse Error
        ,color |-> color
        \* Put additional trace expressions here; examples:
        \* ,x |-> ~y'
        \* ,e |-> ENABLED ActionName
    ]
=============================================================================

---- MODULE TTraceTraceDef ----
EXTENDS Toolbox, TLC, EWD840

def_ov_16088811171232000 == 
    <<
    ([color |-> (0 :> "white" @@ 1 :> "white" @@ 2 :> "white"),tpos |-> 0,tcolor |-> "black",active |-> (0 :> FALSE @@ 1 :> FALSE @@ 2 :> FALSE)]),
    ([color |-> (0 :> "white" @@ 1 :> "white" @@ 2 :> "white"),tpos |-> 2,tcolor |-> "white",active |-> (0 :> FALSE @@ 1 :> FALSE @@ 2 :> FALSE)]),
    ([color |-> (0 :> "white" @@ 1 :> "white" @@ 2 :> "white"),tpos |-> 1,tcolor |-> "white",active |-> (0 :> FALSE @@ 1 :> FALSE @@ 2 :> FALSE)]),
    ([color |-> (0 :> "white" @@ 1 :> "white" @@ 2 :> "white"),tpos |-> 0,tcolor |-> "white",active |-> (0 :> FALSE @@ 1 :> FALSE @@ 2 :> FALSE)])
    >>
=============================================================================
```

**Comment by alygin** [comment]
@lemmy, I see the parse error in EWD840_TE:

<img width="396" alt="изображение" src="https://user-images.githubusercontent.com/2101250/103519848-49c30080-4e87-11eb-9fe4-619016739279.png">

Doesn't your instance of the extension show it? If not, could you, please, run the `Parse module` command and provide the contents of the SANY output channel?

As for the TTraceTraceDef module, yes, that's an error, I'll have a look at it. Thanks for the report!

**Comment by alygin** [comment]
Hm, acutally, it also finds the error when I parse TTraceTraceDef.tla. But the error is in another file (EWD840_TE.tla), so it doesn't expose itself in the TTraceTraceDef module, but I can see the error and I can jump to it:

<img width="1064" alt="изображение" src="https://user-images.githubusercontent.com/2101250/103542275-c36de500-4ead-11eb-8d9d-aff3d960b2c0.png">

Do you mean this behavior when you say that the errors are not detected? Or does the extension behaves differently for you?

Probably, it would make sense to add a file-level error to the parsed module even if the real error is in another file.

**Comment by lemmy** [comment]
@alygin I should have been more explicit in https://github.com/alygin/vscode-tlaplus/issues/185#issue-774666389.  Don't create three separate files--only create one file `EWD840_TTrace_1608881116.tla` with three modules.  It requires a recent TLC nightly.

**Comment by alygin** [comment]
The fix is available in the [v1.5.3-alpha.2 pre-release](https://github.com/alygin/vscode-tlaplus/releases/tag/v1.5.3-alpha.2).

**Comment by alygin** [comment]
The fix is released in [v1.5.3](https://github.com/alygin/vscode-tlaplus/releases/tag/v1.5.3).
